### PR TITLE
Add Parameter Loss

### PR DESCRIPTION
**Pull request by masa-su**
Parameter Lossの追加．
引数としてinput_varに対応する文字列を指定．
estimateやtrain，testにおいて，値を渡すことで評価可能．
係数をアニーリングするときに使えると思います．

```python
from pixyz.losses import StochasticExpectationLoss, KullbackLeibler, Parameter

kl = KullbackLeibler(p, prior)
exp = StochasticExpectationLoss(p, q)
lmb = Parameter("lmb")

loss = (-exp + lmb * kl).mean()
print(loss)
>> mean(-(E_p(z|x)[log q(y|z)]) + lmb * KL[p(z|x)||r(z)])

loss.evaluate({"x": x, "y": y, "lmd": 1e-3})
```

### CODE REVIEW

**Comment by masa-su** [comment]
Loss周りの構成やクラス名を変更．合わせてdocsも修正．
